### PR TITLE
Add config for maxFrameSize

### PR DESCRIPTION
**Pull request by riyafa**

### CODE REVIEW

File: components/org.wso2.transport.http.netty/src/test/java/org/wso2/transport/http/netty/util/server/websocket/WebSocketRemoteServerInitializer.java [hunk 1]
```diff
@@ -43,13 +43,13 @@ public WebSocketRemoteServerInitializer(SslContext sslCtx, String subProtocols)
...
     @Override
-    public void initChannel(SocketChannel ch) throws Exception {
+    public void initChannel(SocketChannel ch) {
         ChannelPipeline pipeline = ch.pipeline();
         if (sslCtx != null) {
             pipeline.addLast(sslCtx.newHandler(ch.alloc()));
         }
         pipeline.addLast(new HttpServerCodec());
-        pipeline.addLast(new HttpObjectAggregator(65536));
+        pipeline.addLast(new HttpObjectAggregator(8192));
```
Comment (arukshani) [inline components/org.wso2.transport.http.netty/src/test/java/org/wso2/transport/http/netty/util/server/websocket/WebSocketRemoteServerInitializer.java]: Shall we define `8192` as a constant and use it?

File: components/org.wso2.transport.http.netty/src/main/java/org/wso2/transport/http/netty/contract/websocket/WebSocketClientConnectorConfig.java [hunk 2]
```diff
@@ -39,6 +39,7 @@
     private int idleTimeoutInSeconds;
     private boolean autoRead;
     private final HttpHeaders headers;
+    private int maxFrameSize = 65536;
```
Comment (Manuri) [inline components/org.wso2.transport.http.netty/src/main/java/org/wso2/transport/http/netty/contract/websocket/WebSocketClientConnectorConfig.java]: Shouldn't this be a final constant?

File: components/org.wso2.transport.http.netty/src/test/java/org/wso2/transport/http/netty/util/server/websocket/WebSocketRemoteServerInitializer.java (hunk 1 above)
Comment (Manuri) [inline components/org.wso2.transport.http.netty/src/test/java/org/wso2/transport/http/netty/util/server/websocket/WebSocketRemoteServerInitializer.java]: Can we make this value also a constant indicating what this means?

File: components/org.wso2.transport.http.netty/src/main/java/org/wso2/transport/http/netty/contract/websocket/WebSocketClientConnectorConfig.java (hunk 2 above)
Comment (riyafa) [reply components/org.wso2.transport.http.netty/src/main/java/org/wso2/transport/http/netty/contract/websocket/WebSocketClientConnectorConfig.java]: It is the default value for this parameter. This parameter is not a constant.

**Pull request merged**